service-discovery/README.adoc

```diff
@@ -31,304 +30,371 @@ WARNING: This content is generated by running npm --prefix .build run generate:a
 
-.Prerequisites
-ifndef::community[]
-* You have a Red Hat account.
-endif::[]
-* You have a running Kafka instance in {product}.
+The RHOAS Operator enables you to expose Kafka instances created in {product-long} to applications in separate OpenShift clusters. When you install the Operator, you can use the RHOAS CLI to inject connection details for a specified Kafka instance to an application running in a given namespace in your OpenShift cluster. With connection between your application and Kafka instance established, you can then work directly with the Kafka instance using standard OpenShift features and APIs.
 
-// Condition out QS-only content so that it doesn't appear in docs.
-// All QS anchor IDs must be in this alternate anchor ID format `[#anchor-id]` because the ascii splitter relies on the other format `[id="anchor-id"]` to generate module files.
-ifdef::qs[]
-[#description]
-Using the {product-long} services in your OpenShift cluster{product-long}.
+When you use the RHOAS CLI to bind a Kafka instance with an application, the RHOAS Operator injects connection details as files into the Pod for your application. By default, the Operator creates the following directory and file structure in the application Pod:
 
-[#introduction]
-Welcome to the {product-long} Service Discovery quick start. In this quick start, you'll learn how to connect your {product} services with https://quarkus.io/[Quarkus^]. Tutorial will connect Kafka with your Application to produce and consume messages from your Kafka instances in {product}.
-endif::[]
+[source, subs="+quotes"]
+----
+/bindings/__<my-kafka-instance>__
+├── bootstrapServers
+├── password
+├── provider
+├── saslMechanism
+├── securityProtocol
+├── type
+└── user
+----
 
-[id="proc-installing-cli_{context}"]
-== Installing required cli tools
+Each file that the Operator injects into the application Pod contains a single connection detail, included in plain text. The connection details that correspond to these files are described below.
 
-RHOAS operator allow you to represent {product} services as first class citizens in your OpenShift environment.
-This enables you to work and integrate with these services using standard OpenShift/Kubernetes features and APIs.
+*bootstrapServers*:: Bootstrap server endpoint for the Kafka instance
+*password*:: Password for connection to the Kafka instance
+*provider*:: Cloud provider for the Kafka instance
+*saslMechanism*:: __Simple Authentication and Security Layer__ (SASL) mechanism used by the Kafka instance for client authentication
+*securityProtocol*:: Protocol used by the Kafka instance to secure client connections
+*type*:: Metadata that identifies the Red Hat OpenShift Application Services (RHOAS) service. For a Kafka instance in {product-long}, this is set to a value of `kafka`.
+*user*:: User name for connection to the Kafka instance
```

Comment:
As agreed let's also add note that this is file type binding and we can also have those values as capitals for environment based bindings

Reply:
Discussed in person with @wtrocki. This section is focused on how the Operator injects credentials. We agreed that it is appropriate to omit this note.